examples/subscription/credit_card_subscription_example.py

```diff
@@ -0,0 +1,118 @@
+# This is just for very basic implementation reference, in production, you should validate the incoming requests and implement your backend more securely.
+import midtransclient
+import datetime
+
+# Initialize core api client object
+# You can find it in Merchant Portal -> Settings -> Access keys
+core_api = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+
+# To use API subscription for credit card, you should first obtain the 1 click token
+# Refer to docs: https://docs.midtrans.com/en/core-api/advanced-features?id=recurring-transaction-with-subscriptions-api
+
+# You will receive saved_token_id after complete 3ds authentication (it also available in the JSON of HTTP notification)
+# Refer to docs: https://docs.midtrans.com/en/core-api/advanced-features?id=sample-3ds-authenticate-json-response-for-the-first-transaction
```

Comment:
why is this grammar not updated? When I told you to fix the description on the readme, intuitively you should also fix this, as the sentence is identical.